policyengine_us/variables/gov/states/ga/tax/adjusted_gross_income/ga_earned_income_eligible.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class ga_earned_income_eligible(Variable):
```

Comment:
should this be the eligibility criteria for one of the exclusions? If so we need to make this a bool

Reply:
No, it's portion of earned income qualified for retirement exclusion. I renamed it.